lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -375,6 +397,33 @@ class LighthouseReportViewer {
       self.opener.postMessage({opened: true}, '*');
     }
   }
+
+  /**
+   * @param {PSIParams} params
+   */
+  _loadFromPSI(params) {
+    const placeholder = find('.viewer-placeholder-inner', document.body);
+    placeholder.classList.add('lh-loading');
+    logger.log('Waiting for Lighthouse results ...');
+    return this._psi.fetchPSI(params).then(response => {
+      logger.hide();
+      placeholder.classList.remove('lh-loading');
+
+      if (!response.lighthouseResult) {
+        if (response.error) {
+          // eslint-disable-next-line no-console
+          console.error(response.error);
```

Comment:
looks like logger.error actually throws (in a settimeout) so don't need to manually log to console

Reply:
fyi that misses out on other properties on `response.error` (only message is sent to logger.error).